{{cookiecutter.project_name}}/apps/users/templatetags/userindicator.py

```diff
@@ -0,0 +1,31 @@
+
+from allauth.account import urls as account_urls
+from django import template
+from django.core.urlresolvers import Resolver404
+from django.core.urlresolvers import resolve
+
+INVALID_URL_NAME = object()
+
+register = template.Library()
+
+
+def _is_account_url(request):
```

Comment:
maybe use https://docs.djangoproject.com/en/2.0/ref/request-response/#django.http.HttpRequest.resolver_match
that would be something like request.resolver_match.app_name == allaout.app.Config.label (pseudo code)